README.md

```diff
@@ -87,11 +87,8 @@ If you set the value of this variable to "true" then the Lambda layer will incre
 
 ### DD_LOGS_INJECTION
 
-By default, the Datadog trace id gets automatically injected into the logs for correlation, if using `console` or a logging library supported for [automatic](https://docs.datadoghq.com/tracing/connect_logs_and_traces/?tab=nodejs#automatic-trace-id-injection)  trace id injection.
-
-See instructions for [manual](https://docs.datadoghq.com/tracing/connect_logs_and_traces/?tab=nodejs#manual-trace-id-injection) trace id injection, if using other logging libraries.
-
-Set the environment variable `DD_LOGS_INJECTION` to `false` to disable this feature.
+Controlls whether or not the request ID is injected into log lines. See [DD_LOGS_INJECTION](#DD_LOGS_INJECTION-environment-variable) under
```

Comment:
@DarcyRaynerDD correct me if i'm wrong. I thought DD_LOG_INJECTION only inject datadog trace id into logs?

Reply:
Okay yep, I misread the other section.